addded the release note for name attribute of details element (#35447)
mdn/content: files/en-us/mozilla/firefox/releases/130/index.md

### HTML

> - The [`name`](/en-US/docs/Web/HTML/Element/details#name) attribute of the `<details>` element now allows the grouping of `<details>` elements, this allows you to create an exclusive accordion, without the use of JavaScript, as only one `<details>` element can be open at a time ([Firefox bug 1856460](https://bugzil.la/1856460) and [Firefox bug 1909613](https://bugzil.la/1909613)).

Reviewer: How about this rephrasing and also splitting the long sentence like so:
```suggestion
- The [`name`](/en-US/docs/Web/HTML/Element/details#name) attribute of the `<details>` element now allows the grouping of `<details>` elements, where only one element within a group can be open at a time. This allows you to create an exclusive accordion without using JavaScript ([Firefox bug 1856460](https://bugzil.la/1856460) and [Firefox bug 1909613](https://bugzil.la/1909613)).
```
